lib/fizzy/trunc_boundaries.hpp

```diff
@@ -51,24 +50,22 @@ template <>
 struct trunc_boundaries<float, int64_t>
 {
     /// The first representable value lower than theoretical -9223372036854775809.
-    static constexpr float lower = -9223373136366403584.0;
-
-    static constexpr float upper = 9223372036854775808.0;
+    static constexpr float lower = -9223373136366403584.0f;
+    static constexpr float upper = 9223372036854775808.0f;
 };
 
 template <>
 struct trunc_boundaries<float, uint64_t>
 {
-    static constexpr float lower = -1.0;
-    static constexpr float upper = 18446744073709551616.0;
+    static constexpr float lower = -1.0f;
+    static constexpr float upper = 18446744073709551616.0f;
 };
 
 template <>
 struct trunc_boundaries<double, int64_t>
 {
     /// The first representable value lower than theoretical -9223372036854775809.
     static constexpr double lower = -9223372036854777856.0;
-
```

Comment:
What is the problem here?

Reply:
Here, nothing, just some cases had an empty line other don't, made that consistent.

Reply:
To separate elements with comments.

Reply:
What do you mean? Doxygen?

Reply:
I understood that it's to show that the comment refers to the next line only, and doesn't to the line after.

Reply:
Yes, but the new way is fine too.